pytest_timeout.py

```diff
@@ -249,10 +244,10 @@ def pytest_exception_interact(node):
 def pytest_enter_pdb():
     """Stop the timeouts when we entered pdb.
 
-    This stops timeouts from triggering when pytest's builting pdb
+    This stops timeouts from triggering when pytest's building pdb
```

Comment:
```suggestion
    This stops timeouts from triggering when pytest's builtin pdb
```

Reply:
fixed this, thanks
And change black 25.9.0 to 25.11.0